### PR TITLE
Update developer guide use cases

### PR DESCRIPTION
**Pull request by Goh-Li-Ting**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -283,32 +283,66 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-    Use case ends.
+**Use case: UC2 - Add a new employee**
 
-**Extensions**
+**MSS**
 
-* 2a. The list is empty.
+1. User requests to add a new employee.
+1. EmployeeManager requests for details of the employee to be added.
```
Comment (Weiennn) [inline docs/DeveloperGuide.md:301]: Should this be condensed to one sentence? e.g. User sends employee data to be added to EmployeeManager

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -283,32 +283,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-    Use case ends.
+**Use case: UC2 - Add a new employee**
 
-**Extensions**
+**MSS**
 
-* 2a. The list is empty.
+1. User requests to add a new employee with employee details.
+1. EmployeeManager adds the employee.
+1. EmployeeManage shows an updated list of employees.
```
Comment (aexolate) [inline docs/DeveloperGuide.md:302]: Minor typo - 'EmployeeManage'

Comment (aexolate) [inline docs/DeveloperGuide.md:302]: Minor typo - "EmployeeManage"

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -283,32 +283,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+**Use case: UC3 - Delete an employee**
 
-    * 3a1. AddressBook shows an error message.
+**MSS**
 
-      Use case resumes at step 2.
+1. User requests to delete an employee with an employee ID.
+1. EmployeeManager deletes the employee.
+1. EmployeeManage shows an updated list of employees.
```
Comment (aexolate) [inline docs/DeveloperGuide.md:332]: Minor typo - "EmployeeManage"

**Review by aexolate — CHANGES_REQUESTED** [review]

**Review by Weiennn — APPROVED** [review]
LGTM

**Pull request merged**